ballerina/init.bal

```diff
@@ -18,7 +18,7 @@ import ballerina/jballerina.java;
 
 function init() {
     setModule();
-    _ = initializeHttpLogs(traceLogConfig, accessLogConfig);
+    _ = initializeHttpLogs(traceLogConsole, traceLogAdvancedConfig, accessLogConfig);
```

Comment:
```suggestion
        TraceLogConfiguration newConfig = traceLogConfig;

        if (!traceLogConfig.console) {
            newConfig.console = traceLogConsole;
        }

        _ = initializeHttpLogs(newTraceLogConfig, accessLogConfig);
```

Reply:
Is this possible?

Reply:
The configurable traceLogConfiguration record is implicitly readonly and fields are also readonly. So assigning `TraceLogConfiguration newConfig = traceLogConfig;` will make newConfig also readonly. So it is giving an runtime error when try to update the field inside the if condition. traceLogConfig.clone() also give the same error.